samples/Tutorials/CSharpIntermediate/CSharpIntermediate/CSharpIntermediate.Game/06_Scenes/LoadScene.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Stride.Core.Mathematics;
+using Stride.Core.Serialization;
+using Stride.Engine;
+using Stride.Input;
+
+namespace CSharpIntermediate
+{
+    public class LoadScene : SyncScript
```

Comment:
Maybe this should be called `ReloadScene`, since we unload the scene and load it back?